src/components/Collapsed.tsx

```diff
@@ -1,4 +1,5 @@
 // external
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 import { h, JSX } from 'preact';
```

Comment:
what's the unused var?

Reply:
I think this is needed for Preact JSX to work properly